conu/utils/probes.py

```diff
@@ -104,10 +110,13 @@ def _run(self):
                     else:
                         raise result
                 elif not (result == self.expected_retval):
+                    logger.info("waiting for process to end")
                     p.join()
+                    logger.debug("process ended, about to start another one")
                     p = Process(target=self._wrapper, args=(fnc_queue, start))
                     p.start()
                     tries += 1
+                    logger.debug("attempt no. %s started", tries + 1)
```

Comment:
since tries starts at 1 there is no need to add 1 to output. The log should be:
"first process started"
"attempt no. 2 started"
...

Reply:
I left it there accidentally, well spotted!